framework/src/transfers/ReporterTransferInterface.C

```diff
@@ -121,3 +139,44 @@ ReporterTransferInterface::getReporterNamesHelper(std::string prefix,
     rnames.emplace_back(obj_name, prefix + rn.getObjectName() + ":" + rn.getValueName());
   return rnames;
 }
+
+void
+ReporterTransferInterface::check(const ReporterName & reporter,
+                                 const FEProblemBase & problem,
+                                 const std::string & method) const
+{
+  if (!problem.getReporterData().hasReporterValue(reporter))
+    _rti_transfer.mooseError("In ",
+                             method,
+                             ": Reporter with the name \"",
+                             reporter,
+                             "\" within app \"",
+                             problem.getMooseApp().name(),
+                             "\" was not found.");
+}
+
+void
+ReporterTransferInterface::check(const ReporterName & from_reporter,
+                                 const ReporterName & to_reporter,
+                                 const FEProblemBase & from_problem,
+                                 const FEProblemBase & to_problem,
+                                 const std::string & method) const
+{
```

Comment:
Could you just call the previous check function twice instead of having this one?

Reply:
👍🏼